src/modules/imageresizer/ui/Models/ResizeOperation.cs

```diff
@@ -136,9 +136,9 @@ public void Execute()
                     encoder.Frames.Add(
                         BitmapFrame.Create(
                             Transform(originalFrame),
-                            originalFrame.Thumbnail,
+                            thumbnail: null, /* should be null, see #15413 */
```

Comment:
Wasn't this one of those changes that fixed one of the faulty images in your contributions to this module??

Reply:
I changed the thumbnail and colorContext with pull request #14475 just because is looked "right" It was not required for the fixes which came which #14475.  ColorContext was reverted with #14867 and this reverts the thumbnail stuff. I also added a comment for the next person who thinks he is smart.